src/SortProgram.java

```diff
@@ -1,56 +1,77 @@
 import java.util.ArrayList;
+import java.util.Collections;
 import java.util.Scanner;
 
 public class SortProgram {
 
-    private Scanner myScanner = new Scanner(System.in);
+    private ArrayList<String> myList = new ArrayList<>();
 
     public void startProgram() {
-        ArrayList myList = new ArrayList();
-        addToList(myList);
-        System.out.println(myList+"\n");
-        do {
-            printBitch("[ADD]\n[REMOVE]\n");
-            String menuInput = myScanner.next();
-            if (menuInput.equals("ADD")) {
-                printBitch("What would you like to add, bitch?");
-                String item = myScanner.next();
-                myList.add(item);
-                printBitch(myList);
-            } else if (menuInput.equals("REMOVE")) {
-                printBitch("What would you like to remove, bitch?");
-                String item = myScanner.next();
-                int idx = 0;
-                while (idx < myList.size()) {
-                    if (myList.get(idx) == item) {
-                        myList.remove(idx);
-                    } else {
-                        ++idx;
-                    }
-                }
-                printBitch(myList);
-            } else {
+        printBitch("\nHello, bitch, gimme something to list!\n");
+        addListItem();
+        displayMenu();
+    }
+
+    private void displayMenu() {
+        System.out.println("\nYour list: " + myList);
+        printBitch("\n[A]DD\n[R]EMOVE\n[S]ORT\n[E]XIT");
+        menuSelection(getUserInput());
+    }
+
+    private void menuSelection(String input) {
+        switch (firstLetter(input)) {
+            case "a":
+                addListItem();
+                displayMenu();
+                break;
+            case "r":
+                removeListItem();
+                displayMenu();
+                break;
+            case "s":
+                Collections.sort(myList);
+                displayMenu();
+                break;
+            case "e":
+                endProgram();
+                break;
+            default:
+                printBitch("Invalid selection, try again, bitch.");
+                displayMenu();
                 break;
```

Comment:
A `switch` statement takes in a variable, and then checks it against one or more "cases". It's important to note that if you don't `break;` at the end of each `case`, your program's flow will continue to the next `case`'s logic within the `switch` scope until it hits a `break;`.